### PR TITLE
Red Hat posgresql-persistent Helm chart.

### PR DESCRIPTION
**Pull request by phracek**
This pull request contains Red Hat postgresql-persistent Helm chart.

The output from verifier is:

```bash
results:
    - check: v1.0/is-helm-v3
      type: Mandatory
      outcome: PASS
      reason: API version is V2, used in Helm 3
    - check: v1.0/chart-testing
      type: Mandatory
      outcome: FAIL
      reason: 'error loading config file "/.kube/config": open /.kube/config: permission denied'
    - check: v1.0/helm-lint
      type: Mandatory
      outcome: FAIL
      reason: |
        Helm lint has failed:  [INFO] Chart.yaml: icon is recommended
        [ERROR] values.yaml: enum items must be unique
        [ERROR] templates/: values don't meet the specifications of the schema(s) in the following chart(s):
        postgresql-persistent:
        enum items must be unique
    - check: v1.1/has-kubeversion
      type: Mandatory
      outcome: FAIL
      reason: Kubernetes version is not specified
    - check: v1.0/not-contains-crds
      type: Mandatory
      outcome: PASS
      reason: Chart does not contain CRDs
    - check: v1.0/required-annotations-present
      type: Mandatory
      outcome: PASS
      reason: All required annotations present
    - check: v1.0/contains-values-schema
      type: Mandatory
      outcome: PASS
      reason: Values schema file exist
    - check: v1.1/images-are-certified
      type: Mandatory
      outcome: FAIL
      reason: |-
        Failed to certify images : Failed to get images, error running helm template : values don't meet the specifications of the schema(s) in the following chart(s):
        postgresql-persistent:
        enum items must be unique
    - check: v1.0/not-contain-csi-objects
      type: Mandatory
      outcome: PASS
      reason: CSI objects do not exist
    - check: v1.0/contains-values
      type: Mandatory
      outcome: PASS
      reason: Values file exist
    - check: v1.0/has-readme
      type: Mandatory
      outcome: PASS
      reason: Chart has a README
    - check: v1.0/signature-is-valid
      type: Mandatory
      outcome: SKIPPED
      reason: 'Chart is not signed : Signature verification not required'
    - check: v1.0/contains-test
      type: Mandatory
      outcome: PASS
      reason: Chart test files exist
```

Please send me what is the problem about, and I will fix it.

### CODE REVIEW

**Comment by dperaza4dustbit** [comment]
@phracek thanks for submitting your chart to make it through the certification process you first have to submit a PR with the owner file see example here: https://github.com/openshift-helm-charts/charts/pull/442. That PR needs to be by itself, see instructions here: https://github.com/openshift-helm-charts/charts/tree/main/docs#preparation

File: charts/redhat/redhat/postgresql-persistent/0.0.1/values.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+config:
+  postgresql_database: testdb
+  postgresql_password: testp
+  postgresql_user: testu
+  port: 5432
+image:
+  tag:
+    - "latest"
+    - "13-el9"
+    - "13-el8"
```
Comment (dperaza4dustbit) [inline charts/redhat/redhat/postgresql-persistent/0.0.1/values.yaml:14]: This duplicate tag is probably the issue with the lint

**Comment by dperaza4dustbit** [comment]
Once the owners file PR is sent rebase this PR and send the rest of the chart. I would run chart verifier before rebasing to make sure you cover most of the issue you see above. I noticed you failed to connect to cluster which is required to be able to run the tests. Also notice you failed the lint check with an enum issue. I added a comment in values file

File: charts/redhat/redhat/postgresql-persistent/0.0.1/values.yaml (hunk 1 above)
Comment (phracek) [reply charts/redhat/redhat/postgresql-persistent/0.0.1/values.yaml:14]: Thanks stupid typo.

**Comment by phracek** [comment]
The OWNERS file is part of pull request https://github.com/openshift-helm-charts/charts/pull/764

**Comment by dperaza4dustbit** [comment]
Notice that after the version 0.0.1 there should be a "src" directory indicating you are submitting source code instead of targz. See example here: https://github.com/openshift-helm-charts/charts/tree/main/charts/redhat/redhat/jenkins/0.0.1/src

**Pull request merged**